test/OpenTelemetry.Exporter.Zipkin.Tests/TestActivityProcessor.cs

```diff
@@ -0,0 +1,69 @@
+﻿// <copyright file="TestActivityProcessor.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using OpenTelemetry.Trace.Export;
+
+namespace OpenTelemetry.Exporter.Zipkin.Tests
+{
+    public class TestActivityProcessor : ActivityProcessor, IDisposable
```

Comment:
Separately, we need to move all these common helpers into a shared project for tests. (Not for this PR)